Test.txt

```diff
@@ -1 +1,3 @@
-This is a test.
\ No newline at end of file
+This is how you change a file.
```

Comment:
And now reply to the second reply (I think)

---
In reply to: [53844945](https://github.com/kivancmuslu/CFTest/pull/1#discussion_r53844945) [](ancestors = 53844945,53844845)